plugins/source/gcp/client/spec.go

```diff
@@ -4,4 +4,6 @@ package client
 type Spec struct {
 	ProjectIDs            []string `json:"project_ids"`
 	ServiceAccountKeyJSON string   `json:"service_account_key_json"`
+	FolderIDs             []string `json:"folder_ids"`
+	FolderRecursionDepth  int      `json:"folder_recursion_depth"`
```

Comment:
let's setup default for `FolderRecursionDepth` via `SetDefault` method

Reply:
Changed to *int.